db/schema.rb

```diff
@@ -454,16 +454,17 @@
   add_index "series_images", ["series_id"], name: "series_images_series_id_fk", using: :btree
 
   create_table "story_distributions", force: :cascade do |t|
+    t.string   "type",            limit: 255
     t.integer  "distribution_id", limit: 4
-    t.integer  "story_id",        limit: 4
+    t.integer  "piece_id",        limit: 4
```

Comment:
But no more required schema changes to the staging DB right?  (I thought this one was already in there when I ran them).